packages/landing/src/web/Block/BlockItem.tsx

```diff
@@ -138,12 +138,14 @@ const renderBlockItem = (props) => {
           </Box>
         )
       case BlockItemTypeEnum.IMAGE:
-        return (
-          <Box {...boxProps}>
-            <Image src={title} {...titleProps} />
-          </Box>
-        )
-      case BlockItemTypeEnum.STORAGE_IMAGE:
+        const isAbsolutePath = !/^public.+/.test(title)
```

Comment:
Leave comment as to why we're testing this. Also check if we can avoid regex in favour of .startsWith()

Reply:
We can try this but also, title can potentially be reactnode so would need to check if string first which adds as much performance hit as a regex would

Reply:
Would regex apply on react node tho?

Reply:
There are a few downsides of using regular expressions (regex) to test a string instead of using the startsWith method:

1. Complexity: Regular expressions are more complex and have a steeper learning curve compared to the startsWith method. If you are not familiar with regex, it may take some time to understand and write the correct pattern.

2. Performance: Regular expressions can be slower compared to using startsWith. This is because regex engines have to analyze the entire string and apply complex pattern matching rules, whereas startsWith simply checks the beginning of the string.

3. Flexibility: While regex provides more flexibility in pattern matching, it can also be a disadvantage. If you only need to check if a string starts with a specific substring, using startsWith is more straightforward and efficient. Regular expressions are better suited for more complex pattern matching scenarios.

4. Readability: Regular expressions can be difficult to read and understand, especially for someone who is not familiar with them. This can make the code harder to maintain and debug in the future.

Overall, using startsWith is generally a better choice when you only need to check if a string starts with a specific substring, as it is simpler, faster, and more readable. Regular expressions are more suitable for complex pattern matching scenarios where more advanced matching rules are required.

Reply:
GPT :)

Reply:
Anyway this is a simple test so would just keep it simple, will only reach for regex as last resort